db.go

```diff
@@ -312,14 +333,33 @@ func (db *DB) retentionCutoff() (b bool, err error) {
 		return false, nil
 	}
 
+	// Size based retention, if MaxBytes is still -1, it has not been set by the user
```

Comment:
nit but the comment isn't accurate anymore. How about "..., if MaxBytes is less than or equal to 0, only the time based retention will be used"?